src/Impostor.Server/Net/Inner/Objects/Components/InnerPlayerPhysics.cs

```diff
@@ -55,6 +55,10 @@ public override async ValueTask<bool> HandleRpcAsync(ClientPlayer sender, Client
                     Rpc19EnterVent.Deserialize(reader, out ventId);
                     break;
 
+                case RpcCalls.ClimbLadder:
+                    Rpc31ClimbLadder.Deserialize(reader, out byte ladderId, out byte lastClimbLadderSid);
+                    return true;
```

Comment:
Why return here? Just break;

Reply:
Because ClimbLadder does not set ventId which is needed for the PlayerVentEvent that is called on line 66. 

I aimed for a minimal diff, should I duplicate the PlayerVentEvent call?

Reply:
Oh, yeah, just duplicate event call